pom.xml

```diff
@@ -301,6 +301,7 @@
         <module>misc</module>
         <module>js</module>
         <module>quarkus</module>
+        <module>operator</module>
```

Comment:
Remove this change, the operator module is included via a profile.

Reply:
This should be taken care of now.

Reply:
@stustison is this ready for another review?

Reply:
@shawkins Yes it should be ready.